.github/workflows/ci.yaml

```diff
@@ -84,20 +85,33 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
-      # since disk-size is currently too-low, we will build only the docker images required for our purposes
+      # since github-actions gives us 14G only, and fill it up with some garbage
+      # we will free up some space for us (~2GB)
+      - name: Freeing some disk space
+        run: docker system prune --all --force
+
       - name: Build
-        run: |
-          make controller dashboard handler-builder-golang-onbuild
+        run: make docker-images
         env:
           NUCLIO_DOCKER_REPO: ${{ env.REPO }}/${{ env.REPO_NAME }}
 
+      # Clean build leftovers
+      - name: Post build cleanup
+        run: docker system prune --force
```

Comment:
It removes dangling images created by the build process. since docker does not have an automatically gc, pruning with `--force` (to auto accept y/N question) will simply remove dangling images.

Reply:
Why do you need to remove them? Once ci is done the whole instance is scratched I assume

Reply:
basically to make enough space for the following command of saving nuclio produces as a zip